[Doc] Fix env path name in `uv` Python env setup instructions

andyjakubowski opened the pull request:
~~Drops the path argument from `uv venv` instructions in favor of default `uv` behavior.~~

**Update:** Per reviewer feedback, this PR now updates the `uv venv` command to use `myenv` as the env path name.

The current instructions say `uv venv vllm`, where `vllm` is the alternative path we’re telling `uv` to use for setting up the Python virtual environment. This throws an error, because the `vllm` directory already exists and is used for Python source code.

```shell
Using CPython 3.12.9 interpreter at: /opt/homebrew/opt/python@3.12/bin/python3.12
Creating virtual environment with seed packages at: vllm
uv::venv::creation

  × Failed to create virtualenv
  ╰─▶ The directory `vllm` exists, but it's not a virtual environment
```

~~We can drop the path argument from the `uv venv` completely — by default, [`uv` will create a `.venv` directory for the project environment](https://docs.astral.sh/uv/concepts/projects/layout/#the-project-environment). This directory won’t get checked into Git because it includes its own match-all `.gitignore`. We already list `.venv` in the root-level `.gitignore`, too.~~

**Update:** This PR originally proposed completely removing the optional env path name from `uv venv`, but we ended up keeping `myenv` as the name for clarity.

<!--- pyml disable-next-line no-emphasis-as-heading -->

andyjakubowski committed:
Remove alternative path from `uv` Python env setup instructions

The current instructions say `uv venv vllm`, where `vllm` is the path we’re telling `uv` to use for setting up the Python virtual environment. This throws an error, because the `vllm` directory already exists and is used for Python source code.

We can drop the path argument from the `uv venv` completely — by default, `uv` will create a `.venv` directory for the project environment. This directory won’t get checked into Git because it includes its own match-all `.gitignore`. We already list `.venv` in the root-level `.gitignore`, too.

Signed-off-by: Andy Jakubowski <hello@andyjakubowski.com>

mgoin reviewed (commented):
Can you just change both to use `myenv`? This change won't work as is since the `source` command hasn't changed

andyjakubowski committed:
Use `myenv` as `uv` env name

Signed-off-by: Andy Jakubowski <hello@andyjakubowski.com>

andyjakubowski commented:
> Can you just change both to use `myenv`? This change won't work as is since the `source` command hasn't changed

@mgoin Thanks, good catch! I've updated the PR to use `myenv` instead of removing the argument entirely. I’ve also updated the PR title and description to reflect this change. Let me know if this looks good now.

andyjakubowski commented:
Hey @mgoin, just a gentle ping that I made the change you suggested so this can be reviewed again!
